lib/grpc/GrpcServer.ts

```diff
@@ -36,13 +36,14 @@ class GrpcServer {
    * Start the server and begin listening on the provided port
    * @param port
    */
-  public listen = (port: number) => {
+  public listen = (host: string, port: number) => {
```

Comment:
Having `port` as the first parameter before `host` seems to be the convention both elsewhere in the code and in places like node's `net` module.